src/components/BookSearch/EmptyList.js

```diff
@@ -0,0 +1,19 @@
+import styles from "../styles/emptyList.module.css";
+import { ReactComponent as NoItemsIcon } from "../../assets/no-items-icon.svg";
+import { Link } from "react-router-dom";
+import AppRoutes from "../../common/AppRoutes";
+
+export default function EmptyList() {
```

Comment:
Empty list jest bardzo generyczne, potrzeba tu większej dokładności - może zmieńmy to na `BookListEmptyView`?